### DISCUSSION TITLE
Fixing/repairing triangle meshes

### DISCUSSION THREAD

**Discussion by jpmaterial** [isl-org/Open3D#5455] [Q&A] [answered]
Triangle meshes can often be broken, i.e. non-watertight: Missing triangles, bad edges, overlapping triangles, and more. Does Open3D offer any functionality to fix these meshes or is there a plan to add that functionality?

**Comment by benjaminum** [comment, marked as answer]
A function for filling holes was added in this PR https://github.com/isl-org/Open3D/pull/5285 . It will be in the next release.
